src/logger.h

```diff
@@ -1,4 +1,28 @@
 #pragma once
+
+#include <fstream>
+
 #include <boost/log/trivial.hpp>
+#include <boost/log/utility/setup/from_stream.hpp>
+#include <boost/log/core/core.hpp>
+#include <boost/log/sources/logger.hpp>
+
+namespace logging = boost::log;
```

Comment:
@AGaliuzov, I don't think that it's good idea to expose boost namespace with such shortcuts. Use full namespaces.
Also I propose to add some project namespace and wrap all functions\classes with it. E.g. `blsb` - boost log send box . Or propose something better, but we need some custom namespace. `LoggerType` also should be wrapped with namesapce.